http/server_test.ts

```diff
@@ -19,14 +19,17 @@ interface ResponseTest {
 const enc = new TextEncoder();
 const dec = new TextDecoder();
 
+type handler = () => void;
```

Comment:
Types should be capitalized.

Also it would be great if we could get CI to fail on warnings.